HISTORY.rst

```diff
@@ -2,8 +2,22 @@
 History
 =======
 
+v3.48.0 (2021-11-11)
+--------------------
+Changes
```

Comment:
We should include a Critical Changes section warning of the upcoming dropping of support for Python 3.6 and 3.7

Reply:
I think something like:

> CumulusCI will be dropping support for Python 3.6 and 3.7 in the next couple of releases. Please ensure you are running Python>3.8 to ensure you won't be impacted by this change.